recipes/polap/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "polap" %}
+{% set version = "0.2.4" %}
+{% set sha256 = "4463d4a71a8a3273108a0ab2dff0a4c37a565c05af6ee17ee83dc62404294caf" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/goshng/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
```

Comment:
Some as in the other PR. You are using R scripts as well, so you? This should be noarch:generic imho

Reply:
Thank you. I need to figure out what you mean. It is somewhat steep learning curve up until now. I have basically zero about making a condo package.

Reply:
As always, I appreciate your efforts of reviewing the recipe.

Reply:
> Some as in the other PR. You are using R scripts as well, so you? This should be noarch:generic imho

You are right about it. I am using R in the script. Let me have time to figure out what the terms are: noarch:generic and imho. Thank you  for your review.

Reply:
I edited the noarch to noarch:generic as recommended. Thank you.